socorro/processor/symbol_cache_manager.py

```diff
@@ -14,11 +14,19 @@
 from configman.converters import class_converter, timedelta_converter
 
 from socorro.lib.datetimeutil import utc_now, UTC
-import pyinotify
+if os.uname()[0] == 'Darwin':
+    # You're on OSX! Avoid pyinotify like the plague
+    class ProcessEvent(object):
+        # Defining a class means we can't define the EventHandler class
+        # without indenting the whole thing in an if-block.
+        pass
```

Comment:
Can we log a warning message too? This is just going to silently not work otherwise right?